components/automate-ui/src/app/components/pending-edits-bar/pending-edits-bar.component.scss

```diff
@@ -3,16 +3,29 @@
 #pending-edits-bar {
   background-color: $chef-primary-bright;
   color: $chef-white;
-  padding: 16px 10px;
   position: relative;
 
+  .normal-container {
+    padding: 16px 10px;
+  }
+
+  .failure-container {
+    padding: 16px 10px;
+    background-color: $chef-critical;
```

Comment:
These two are in a new child div of `pending-edits-bar`, which starts with the default blue background. If it is a normal-container, that does not change. But if it is a failure-container, that completely overlays the blue background with the pink background.